### ISSUE TITLE
New Token submission

### ISSUE THREAD

**Issue by jtakalai** [iotubeproject/ioTube#282]
# ioTube Token submission

Please read the token submission guide here: https://docs.iotube.org/introduction/token-submission

## Team / Developer name
[Streamr Network AG](https://streamr.network/)

## Token Symbol
DATA

## Token Decimals
18

## Token Name in full
Streamr

## Link to the token icon
![$DATA token symbol](https://github.com/user-attachments/assets/7c72e529-576e-4a84-b19e-0aa4d93c2d69)

Can also be downloaded from https://streamr.network/design under "Media Kit", then `Streamr Brand Assets/Svg/Marks and Symbols/Token symbols/$DATA new token symbol.svg` inside the zip file.

## Short description of your project
The Streamr Network is a peer-to-peer network for publishing and subscribing to data in real-time. Applications use it for decentralized messaging, for example sharing data across applications or broadcasting real-time state changes to large audiences. The decentralized nature of the system makes the data transport scalable, robust, secure, tamper proof, and censorship resistant.

## What is the blockchain (source blockchain) that your token is issued on

* [x] Ethereum  
* [ ] Binance Smart Chain
* [ ] Polygon
* [ ] IoTeX
* [ ] Solana

## What blockchains (destination blockchain) would you like your token is bridged to

* [ ] Ethereum  
* [ ] Binance Smart Chain
* [ ] Polygon
* [x] IoTeX
* [ ] Solana 

(Please check https://docs.iotube.org/introduction/token-submission#integration-fee for integration fees of different blockchains)

## Token native contract address explorer link

Address: [0x8f693ca8D21b157107184d29D398A8D082b38b76](https://etherscan.io/address/0x8f693ca8d21b157107184d29d398a8d082b38b76)

## Your project websites

- https://streamr.network/

## Your social channels

- Telegram: https://t.me/streamrofficial
- X: https://x.com/streamr
- Discord: https://discord.gg/gZAm8P7hK8
- Website: https://streamr.network/

## Verification of the token

Please provide the verification of your token in one of the following ways.

* [x] Token address on your website. Please specify the link.
  * https://streamr.network/data-token
  * Under "Add DATA to your wallet", click "Add token"). We're looking forward to adding IoTeX there ;)
* [x] CMC / Coingecko profile link with token contract address and link to website/social
  * https://coinmarketcap.com/currencies/streamr/ ("Website" button points to https://streamr.network/)

## Send test tokens 

Please send a few test tokens ($1-$10 worth) to our wallet address: `0x571e603e3B5c82Bd40998C538603dF4fb42bB051`.
* [x] Sent 90 DATA (~$3), see [transaction receipt](https://etherscan.io/tx/0xad2fbff5b62f16f1a5511e9912f2704752bc8c3626b8c352f5bdb9b632ad9a8a)

## Integration fee 
Please refer to https://docs.iotube.org/introduction/token-submission for the full list of integration fees. 

Please send the payment to `0x571e603e3B5c82Bd40998C538603dF4fb42bB051` or `io12u0xq03mtjpt6sye33fcvq7lf76zhvz3yvt8q0` and post your txn link here.
* [x] Sent 0.07 ETH (~$230), see [transaction receipt](https://etherscan.io/tx/0x94e1b233b48e0d547edd9cbb75389223a364def45ea0662e7e42e786a3a9851c)

**Comment by jtakalai** [comment]
There already exists DATA token that is bridged from Polygon: https://iotexscan.io/address/0x1ae24d4928a86faaacd71cf414d2b3a499adb29b#code

Is it possible to bridge both from Mainnet and Polygon? The "original" does live in Mainnet. If that's not possible, it's okay to close this ticket.

**Comment by guo** [comment]
Shall we have a tg chat and discuss the details?

**Comment by jtakalai** [comment]
Yes, please! After all, it's maybe not even super useful if another ERC20 token is bridged; ideally we'd get ERC667 (transferAndCall) functionality in there as well. Let's discuss in TG! My username is @jtakalai
